Standards/scs-0200-v1-sonobuoy-kaas-conformance-tests.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Sonobuoy - KaaS conformance test framework
+type: Decision Record
+status: Draft
+track: KaaS
+---
+
+## Introduction - Motivation
+
+With the [k8s-cluster-api-provider][k8s-api], the SCS provides a tool to generate
+and manage k8s clusters on top of its IaaS infrastructure. As part of
+the application, [Sonobuoy][sonobuoy] is used as a test suite to execute the
+official [Kubernetes e2e tests][k8s-e2e-tests]. Future and existing conformance
+tests derived from SCS standards are perhaps part of these tests and could therefore
+be provided by running the integrated [Sonobuoy e2e test plugin][e2e test plugin].
```

Comment:
I think words like "perhaps" should be avoided. Furthermore this reads like you expect our conformance tests could be part of the *upstream* Kubernetes E2E tests at some point, is this intended?

What sounds more logical to me is the following: we expect that current and future tests derived from SCS standards will benefit from (or even require) the testing mechanism(s) that Sonobuoy provides.

Reply:
Yeah that sounds better, I replaced that in the latest commit.